.ci_support/travis_test_backwards.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# # Conda fix
+# eval "$(conda shell.bash hook)"
+# conda activate root
+# env
+
+pip install --force --ignore-installed --no-deps pyiron==0.2.16
+for t in tests/backwards/*save.py; do
```

Comment:
```suggestion
for t in $(ls tests/backwards/*save.py); do
```
I do not see any print out so I have the feeling it might be another shell which travis is using.

Reply:
No, running it locally also doesn't produce any output except 
```
job exists already and therefore was not created!
```
just as on the travis pipeline.  (That's on loading, so there's a probably a mild bug in `Project.load`, but I didn't get to look deeply into it.)

Reply:
There should at least be a print out that the job was save in the database. And sphinx can not be executed without installing it and the installation is currently missing - as highlighted above. To test it you can also add print commands.

Reply:
The sphinx installation is part of `travis_setup_notebooks.sh`, right? That's in c35b226.  I've checked that it is written to hdf and the (sqlite) database on my machine after running the save script, but you're right it should give a message.  That seems to be an unrelated bug then.

Reply:
If you add a simple print message to the script we should see that they are currently not executed. The message that you get:
```
job exists already and therefore was not created!
```
Means the job exists locally already - maybe from a previous test.

Reply:
Ok I was wrong, the prints are shown nicely. Everything is fine from my perspective.